### DISCUSSION TITLE
Autoplay attribute of video tag doesn't work

### DISCUSSION THREAD

**Discussion by yerassyl02-y** [fastapi/fastapi#4580] [Questions] [answered]
```
<video
          loop
          muted
          autoplay
          :controls="false"
          src="@/assets/videos/video.mp4"
          type="video/mp4"
        ></video>
```

Hi gyus!

Here I have the video in my Nuxt project. Here I have autoplay attribute and in desktop screen resolution it works correctly.
But when I checked it in my phone, I encountered with several problems.

1.Video not playing until I click to it.
2.When i click it, i get the video to fullscreen

How can I remove the click and make the video autoplayed in small screen resolutions?

Also I tried to create a `source` tag and insert src here, but it doesn't work anyway

**Comment by tiangolo** [comment, marked as answer]
It seems this is a frontend question, not really related to FastAPI.

I would suggest you ask somewhere else, maybe on StackOverflow. But first, write a self-contained example reproducing the problem. Try with pure vanilla HTML and a local video file, check if it works, and then incrementally update from the simplest thing until you find the point where things break.
